.github/workflows/conformance.yml

```diff
@@ -95,7 +108,7 @@ jobs:
       - name: Install Consul
         working-directory: "consul-api-gateway/internal/testing/conformance"
         run: |
-          helm install --values ./consul-config.yaml consul $GITHUB_WORKSPACE/consul-k8s/charts/consul --create-namespace --namespace=consul
+          helm install --values ./consul-config.yaml consul hashicorp/consul --version ${{ matrix.config.consul-k8s-version }} --set global.image=${{ matrix.config.consul-image }} --set global.imageEnvoy=${{ matrix.config.envoy-image }} --create-namespace --namespace=consul
```

Comment:
cloning `main` has been breaking frequently, and cloning the tag when we can just install from the repo doesn't make sense